src/app/component/auth/components/sign-up/sign-up.component.ts

```diff
@@ -176,4 +193,37 @@ export class SignUpComponent implements OnInit {
       panelClass: 'custom-dialog-container',
     });
   }
+  
+  public getErrorMessage({ errors }: FormControl, tag: string) {
```

Comment:
validators can be moved to a separate file.

Reply:
Or why It's public?

Reply:
fixed